examples/main/main.cpp

```diff
@@ -215,6 +224,8 @@ int main(int argc, char ** argv) {
                 __func__, n_ctx_train, n_ctx);
     }
 
+    LOG_TEE("%s: chat template example: %s\n", __func__, llama_chat_format_example(model, params.chat_template).c_str());
```

Comment:
@ggerganov FYI I also log an example of the formatted chat (like in `server` example), could be useful when user report bugs related to quality of generation. Do you think it's good to leave it in `main`?